src/extension.ts

```diff
@@ -5,6 +5,12 @@ import { startLinting } from "./linter";
 import { config } from "./configuration";
 import * as client from "./client";
 
+const storeRegex = /\/nix\/store\/.{32}-[^/]*\/([^:]*)/g;
```

Comment:
can we make this store-path a configurable value?

Reply:
may there is already nix  env variable or nix conf file having the path? so when you run nix you run from that path and may be extensions can access it?

Reply:
There's [`NIX_STORE_DIR`](https://nixos.org/manual/nix/stable/command-ref/env-common.html#env-NIX_STORE_DIR); we could check that and then fallback to `/nix/store`.